### PR TITLE
Azure credits program page

### PR DESCRIPTION
**Pull request by fossygirl**
Working on text for Azure credits page

### CODE REVIEW

**Review by jeffwilcox — APPROVED** [review]
Sounds good. Is this a work-in-progress, or would you like this merged?

**Comment by fossygirl** [comment]
Just a WIP. I'll add a Do Not Merge addendum.

**Comment by richlander** [comment]
Best practice is to use a draft PR for WIP work that you don't want merged.

File: azure-credits.html [hunk 1]
```diff
@@ -18,7 +18,8 @@ <h1 class="h2">Azure credits for open source projects</h2>
         <div class="wrapper my-6 py-4">
             <div class="col-md-10 col-lg-7 mx-auto">
                 <h3 class="h4 font-weight-400 mb-4">
-                  This is some introductory text.
+                  We are offering Azure credits to open source projects for their use for testing, builds, and other infrustructure support needs. Projects must
```
Comment (richlander) [inline azure-credits.html:21]: Would be good to clarify (here or elsewhere) how long the aware lasts and if it is renewable.

**Comment by Dreamypops33** [comment]
<h4>="Jason Howard"

**Pull request merged**